.github/workflows/ci.yml

```diff
@@ -223,13 +227,132 @@ jobs:
           if [ -d ~/.ccache ]; then mv ~/.ccache .; fi
           mkdir build
           cd build
-          cmake -DUSE_SHARED_LIBS= -DSTRICT_GSL_CHECKS=AUDIT -DCI_BUILD=ON -DDISABLE_JEMALLOC=ON -DENABLE_AWS=ON -DENABLE_LIBRDKAFKA=ON -DENABLE_MQTT=ON -DENABLE_AZURE=ON -DENABLE_SQL=ON \
-              -DENABLE_SPLUNK=ON -DENABLE_GCP=ON -DENABLE_OPC=ON -DENABLE_PYTHON_SCRIPTING=ON -DENABLE_LUA_SCRIPTING=ON -DENABLE_KUBERNETES=ON -DENABLE_TEST_PROCESSORS=ON -DENABLE_PROMETHEUS=ON \
-              -DDOCKER_BUILD_ONLY=ON -DDOCKER_CCACHE_DUMP_LOCATION=$HOME/.ccache ..
+          cmake ${DOCKER_CMAKE_FLAGS} ..
           make docker
+      - name: Save docker image
+        run: cd build && docker save -o minifi_docker.tar apacheminificpp:0.13.0
```

Comment:
It wasnt super simple to go with the suggested route, because this versioning-scheme  is used/anticipated in multiple places.
Found a simpler method though: [we can simply extract the version from CMake](https://github.com/apache/nifi-minifi-cpp/pull/1538/commits/6a7cd955d9c0e37815ee27ab2dfcf149a0b932cb).